swift-sdk/AuthManager.swift

```diff
@@ -51,16 +56,43 @@ class AuthManager: IterableInternalAuthManagerProtocol {
     
     func retrieveAuthToken() {
         authToken = localStorage.authToken
+        
+        queueAuthTokenExpirationRefresh(authToken)
     }
     
     // MARK: - Private/Internal
     
+    private static let refreshWindow = 60
```

Comment:
Consider allowing setting this variable in constructor in addition to a default value.

Reply:
That will also have the added benefit to being able to unit test it.